shell/config/product/multi-cluster-apps.js

```diff
@@ -65,7 +54,6 @@ export function init(store) {
   });
 
   basicType([
-    'mc-apps',
     'mc-catalogs',
```

Comment:
Should `Catalogs` also be removed, not sure they're used anywhere outside of mc apps...  or do we need the deprecated process there as well?